docs/contributors.rst

```diff
@@ -28,19 +28,11 @@ Help with hacking!
 ==================
 
 First step is to set up Kitsune so you can run it and hack on it. For
-that, see the :ref:`hacking-howto-chapter`.
```

Comment:
It would be probably helpful to have a section on where to file bugs for this project and a link to Bugzilla.